### PR TITLE
#586 Fix connector config cannot be saved with the new API issue

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

required by #586

### CODE REVIEW

**Comment by konekoya** [comment]
Fix connector forms not able to save config issue due to recent API changes

/cc @HarryChiang143

File: ohara-manager/client/src/constants/pipelines.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 export const CONNECTOR_TYPES = {
```
Comment (chia7712) [inline ohara-manager/client/src/constants/pipelines.js]: Do we start to generate connector form by metadata?

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/FtpSource.js [hunk 2]
```diff
@@ -161,13 +161,7 @@ class FtpSource extends React.Component {
     const { fileEncodings, tasks } = this.state;
 
     if (result) {
-      const {
-        schema = '[]',
-        name = '',
-        state,
-        configs,
```
Comment (chia7712) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/FtpSource.js]: oh...I overlook "configs" when keeping the compatibility...

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/Connectors/FtpSource.js]: No worries :) I'm fixing it in the frontend

File: ohara-manager/client/src/constants/pipelines.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/constants/pipelines.js]: You mean the official connectors? Not yet 😅

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Pull request merged**